src/window.rs

```diff
@@ -159,7 +159,7 @@ pub struct WindowAttributes {
     pub resize_increments: Option<Size>,
     pub content_protected: bool,
     pub window_level: WindowLevel,
-    pub parent_window: Option<RawWindowHandle>,
+    pub parent_window: SendSyncWrapper<Option<RawWindowHandle>>,
```

Comment:
I think we should either move this out of `WindowAttributes` into `PlatformSpecificWindowBuilderAttributes` or mark it as private. This would also solve the issue with the visibility of `SendSyncWrapper` pointed out above.

Reply:
Alternatively, we could just wrap `WindowAttributes` in `SendSyncWrapper` ...